src/algorithms/reco/JetReconstruction.h

```diff
@@ -0,0 +1,62 @@
+// SPDX-License-Identifier: LGPL-3.0-or-later
+// Copyright (C) 2023 Derek Anderson, Zhongling Ji
+
+#ifndef EICRECON_JETRECONSTRUCTION_H
+#define EICRECON_JETRECONSTRUCTION_H
+
+// standard c includes
+#include <vector>
+// DD4hep includes
+#include <spdlog/spdlog.h>
+#include <DD4hep/DD4hepUnits.h>
+#include <edm4hep/utils/kinematics.h>
+#include <edm4eic/MutableReconstructedParticle.h>
```

Comment:
The MutableReconstructedParticle.h needs to be included in the .cc file.
```suggestion
#include <edm4eic/ReconstructedParticle.h>
```

Reply:
Gotcha! So "ReconstructedParticle.h" in the .h file and "MutableReconstructedParticle.h" in the .cc file?